package.json

```diff
@@ -1,17 +1,27 @@
 {
   "name": "microverse-react-bookstore",
   "version": "0.1.0",
+  "homepage": "https://benawi.github.io/Microverse-React-Bookstore",
   "private": true,
```

Comment:
**[OPTIONAL]**
- [ ] It would be nice if you remove this `"homepage": "https://benawi.github.io/Microverse-React-Bookstore",` because when we run npm start then it opens a blank page and we need to manually change the URL to get the required page 😨 Kindly remove this homepage to fix this issue 😊